Pulling most up-to-date changes from Pietro's storybook (#44)
bbc/digital-paper-edit-storybook: CONTRIBUTING.md

## Pull requests and branching

> 1. [Feel free to start by raising an issue](https://github.com/pietrop/digital-paper-edit-storybook/issues/new?template=feature_request.md) so that others can be aware of areas where there is active development, and if needed we can synchronies the effort.

Author: ```suggestion
1. [Feel free to start by raising an issue](https://github.com/bbc/digital-paper-edit-react-components/issues/new?template=feature_request.md) so that others can be aware of areas where there is active development, and if needed we can synchronies the effort.
```

2. [Fork the repo](https://help.github.com/articles/fork-a-repo/)

3. Before working on a feature **always** create a new branch first. Createa a branch with a meaningful name.
4. Branches should be short lived - consider doing multiple PR breaking down functionalities as opposed to one big change.

> 5. If you've added code that should be tested, add tests, if you need help with automated testing, feel free to raise an [issue](https://github.com/pietrop/digital-paper-edit-storybook/issues/new?template=feature_request.md).

Author: ```suggestion
5. If you've added code that should be tested, add tests, if you need help with automated testing, feel free to raise an [issue](https://github.com/bbc/digital-paper-edit-storybook/issues/new?template=feature_request.md).
```

6. Ensure the test suite passes.
7. Make sure your code lints.

> 8. If you've changed APIs, consider [updating the documentation](https://github.com/pietrop/digital-paper-edit-storybook#documentation) and QA Testing docs.

Author: ```suggestion
8. If you've changed APIs, consider [updating the documentation](https://github.com/bbc/digital-paper-edit-storybook#documentation) and QA Testing docs.
```

- [ ] Ensure test suite passes (`npm run test`)
- [ ] Make sure your code lints. (`npm run lint`)
- [ ] consider re-factoring for code quality and readability

> - [ ] Update documentation and QA docs where appropriate - [see updating the documentation](https://github.com/pietrop/digital-paper-edit-storybook#documentation)

Author: ```suggestion
- [ ] Update documentation and QA docs where appropriate - [see updating the documentation](https://github.com/bbc/digital-paper-edit-storybook#documentation)
```
